aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,21 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.
+2. To turn a feature on or off also consider using a [toggle button](https://design-system.hpe.design/components/button#toggle-buttons)
```

Comment:
Not sure if this bullet is correctly placed. Seems out of line with the other items which are use cases specifically for Checkboxes outside of a FormField.